### PR TITLE
apply maximum line width of 120 characters

### LINKED ISSUE #593: Make R/ conform to `line_length_linter(120)`
Part of #584 

- [ ] set `line_length_linter` to 120 in `.lintr`
- [ ] run `lintr::lint_package()` to find all lints
- [ ] fix all lints

### PR DESCRIPTION
**Pull request by MichaelChirico**
Closes #593 

> set line_length_linter to 120 in .lintr

That's already done for this linter right?

### CODE REVIEW

File: R/lint.R
Comment (MichaelChirico) [inline R/lint.R:250]: LMK the preferred way to write long function signatures. I'm partial to

```
foo <- function(
  args...
) {
```

But I often see

```
foo <- function(arg1,
                arg2,
                arg2,
                ...) {
```

**Comment by AshesITR** [comment]
Once the kick-off PR is merged (currently waiting for CI to pass), the line length will be 160.

File: R/lint.R
Comment (AshesITR) [reply R/lint.R:250]: The rest of the codebase, e.g. `lint_dir` and `lint_package` use the latter style. We should be consistent I think.

File: R/get_source_expressions.R
Comment (AshesITR) [inline R/get_source_expressions.R:172]: Can we move the `NULL` check to `safe_parse_to_xml` as well?

File: R/closed_curly_linter.R
Comment (AshesITR) [inline R/closed_curly_linter.R:41]: Not quite sure, but I think the default tidyverse style doesn't encourace trailing `<-`.
Would one line break after `||` be sufficient for the 120-char limit?

File: R/open_curly_linter.R
Comment (AshesITR) [inline R/open_curly_linter.R:37]: Same as in `closed_curly_linter.R`.

**Review by AshesITR — CHANGES_REQUESTED** [review]
Thanks for helping with the delinting process.

File: R/get_source_expressions.R
Comment (MichaelChirico) [reply R/get_source_expressions.R:172]: OK. I only didn't initially because the other usage doesn't have the `NULL` check (I assume because it's always non-`NULL` there, but wasn't sure enough to proceed).

File: R/closed_curly_linter.R
Comment (MichaelChirico) [reply R/closed_curly_linter.R:41]: I don't see anything in the style guide:

 - long lines: https://style.tidyverse.org/syntax.html#long-lines
 - assignment: https://style.tidyverse.org/syntax.html#assignment-1
 - long lines again: https://style.tidyverse.org/functions.html#long-lines-1

Locally I have `xml2`, `dplyr`, and `roxygen2`, only `roxygen2` has one line ending in `<-`:

```
grep -Er "<-$" dplyr/R | wc -l
```

So at a glance it looks avoided; will change.

**Review by AshesITR — DISMISSED** [review]
LGTM, thanks!

**Comment by AshesITR** [comment]
Codecov could be made to pass by adding a new test for `expect_lint` that expects an error when giving a regex with capture groups.

File: R/actions.R
Comment (MichaelChirico) [inline R/actions.R:8]: should we nocov this?

Comment (AshesITR) [reply R/actions.R:8]: Would be fine with it. Could also add a test similar to `trim_output()`, shouldn't be too hard because it only `cat()`s, so `testthat` can capture and check its output.

Comment (MichaelChirico) [reply R/actions.R:8]: For some reason I assumed we wouldn't be able to get this working without actually being on GHA. But we can imitate that by messing with env & options, so I added a test.

File: tests/testthat/test-actions.R
Comment (AshesITR) [inline tests/testthat/test-actions.R:7]: `=` -> `<-`

**Review by AshesITR — APPROVED** [review]

**Pull request merged**
